### PR TITLE
[Merged by Bors] - chore: restore `simp`s that used to work

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mo271** [comment]
!bench

**Comment by mo271** [comment]
I added `!bench` although I don't really expect any regression in speed, but just to be sure...

**Comment by grunweg** [comment]
There's some overlap with #12128 (but not too bad). In any case, thanks for doing this also!

File: Mathlib/Analysis/NormedSpace/AffineIsometry.lean [hunk 1]
```diff
@@ -847,10 +847,7 @@ theorem AffineMap.continuous_linear_iff {f : P →ᵃ[𝕜] P₂} : Continuous f
       (AffineIsometryEquiv.vaddConst 𝕜 <| f default).toHomeomorph.symm ∘
         f ∘ (AffineIsometryEquiv.vaddConst 𝕜 default).toHomeomorph := by
     ext v
-    -- Porting note: was just `simp`
-    simp only [(AffineIsometryEquiv.coe_toHomeomorph), Function.comp_apply, (AffineMap.map_vadd),
-      eq_self_iff_true, (AffineIsometryEquiv.toHomeomorph_symm),
-      (AffineIsometryEquiv.coe_vaddConst), (vadd_vsub), (AffineIsometryEquiv.coe_vaddConst_symm)]
+    simp
```
Comment (grunweg) [inline Mathlib/Analysis/NormedSpace/AffineIsometry.lean:850]: I wonder if this one will make a performance difference. Benchmarking will tell.

File: Mathlib/Analysis/NormedSpace/AffineIsometry.lean [hunk 2]
```diff
@@ -863,10 +860,7 @@ theorem AffineMap.isOpenMap_linear_iff {f : P →ᵃ[𝕜] P₂} : IsOpenMap f.l
...
```
Comment (grunweg) [inline Mathlib/Analysis/NormedSpace/AffineIsometry.lean:863]: Likewise here. A mere `simp` is nicer to read in any case.

**Review by grunweg — APPROVED** [review]
Thanks for doing this, looks good to me!

**Comment by mo271** [comment]
> There's some overlap with #12128 (but not too bad). In any case, thanks for doing this also!

Haha, that was almost at the same moment...
Happy to do rebasing in either order..

**Comment by grunweg** [comment]
*If* you want to continue along this vein, [this branch](MR-porting-notes-simp) of mine contains a few more changes that I haven't cleaned up yet (and my three porting notes PRs of this morning). I will eventually get to these. But if you want (no pressure!!), feel welcome to crib/clean those up.

**Comment by mo271** [comment]
> _If_ you want to continue along this vein, [this branch](MR-porting-notes-simp) of mine contains a few more changes that I haven't cleaned up yet (and my three porting notes PRs of this morning). I will eventually get to these. But if you want (no pressure!!), feel welcome to crib/clean those up.

I have no commits in the vein planned, so please go ahead without me

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**